### PR TITLE
Create basic article 8_fragments.md

### PR DESCRIPTION
**Pull request by cadamini**
## Issue Fixed

Closes https://github.com/utkarsh006/Eazy-Android/issues/24 (not sure though, see below)

## Changes Proposed

I created a basic fragment article with information from https://developer.android.com/guide/fragments and the subordinated articles. 

## Check List ( Tick all the applicable boxes )

- [x] My Code follows the code style of this project.
- [x] The title of My PR is a short description of the requested changes.
- [x] Article/Blog added in Article Folder.
- [x] I have starred this repo.

## Note to Reviewers 

As I have said in the issue, I have no clue about Android development, so my only chance was to copy and paste text with links to the refs.

If this is not ok or sufficient, feel free to close the PR.

### CODE REVIEW

File: Articles/8_fragments.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+A Fragment represents a reusable portion of your app's UI. A fragment defines and manages its own layout, has its own lifecycle, and can handle its own input events. Fragments cannot live on their own--they must be hosted by an activity or another fragment. The fragment’s view hierarchy becomes part of, or attaches to, the host’s view hierarchy.
```
Comment (cadamini) [inline Articles/8_fragments.md:1]:
```suggestion
# Fragments

A Fragment represents a reusable portion of your app's UI. A fragment defines and manages its own layout, has its own lifecycle, and can handle its own input events. Fragments cannot live on their own--they must be hosted by an activity or another fragment. The fragment’s view hierarchy becomes part of, or attaches to, the host’s view hierarchy.
```

File: Articles/8_fragments.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+To properly react to user events, or to share state information, you often need to have channels of communication between an activity and its fragments or between two or more fragments. To keep fragments self-contained, you should not have fragments communicate directly with other fragments or with its host activity.
+
+The Fragment library provides two options for communication: a shared `ViewModel` and the Fragment Result API. The recommended option depends on the use case. o share persistent data with any custom APIs, you should use a `ViewModel`.
+
+See https://developer.android.com/guide/fragments/communicate for details.
+
+## More insights on how to use with fragements
+
+Fragements are interconnected with activities. Learn more about the introductionary words above on https://developer.android.com/guide/fragments.
```
Comment (cadamini) [inline Articles/8_fragments.md:57]:
```suggestion
Fragments are interconnected with activities. Learn more than the introductory words above at https://developer.android.com/guide/fragments.
```

File: Articles/8_fragments.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# Fragments
+
```
Comment (cadamini) [inline Articles/8_fragments.md:2]:
```suggestion

*(all texts in the article from https://developer.android.com/guide/fragments)*  

```

**Comment by utkarsh006** [comment]
I think you are right, it's better to close this PR. Anyways, thanks for your contribution to the repo till now.

**Comment by nishkarsh800** [comment]
@utkarsh006 Can u please assign this issue to me

**Pull request closed without merging**